main/modes/mode_tunernome.c

```diff
@@ -68,7 +72,7 @@
 // #define MAX(X, Y) ( ((X) > (Y)) ? (X) : (Y) )
 
 #define NUM_SEMITONES 12
-#define NUM_TSIGS 8
+#define NUM_TSIGS 7
```

Comment:
This could be replaced with `(sizeof(tSigs) / sizeof(tSigs[0]))` so you don't have to declare or track it separately.

Reply:
Same goes for any time you're looking at the length of some `const` array, like `NUM_GUITAR_STRINGS`, `NUM_VIOLIN_STRINGS`, `NUM_UKULELE_STRINGS`, or `NUM_BANJO_STRINGS`. There may be others, I'll stop pointing them out

Reply:
`NUM_SEMITONES` ok I'm done

Reply:
Implemented! Except for `NUM_SEMITONES`